src/ol_dbt/models/staging/mitlearn/stg__mitlearn__app__postgres__learning_resources_userlist.sql

```diff
@@ -0,0 +1,18 @@
+with source as (
+    select *
+    from {{ source('ol_warehouse_raw_data', 'raw__mitlearn__app__postgres__learning_resources_userlist') }}
+)
+
+, cleaned as (
+    select
+        id as userlist_id
+        , title as userlist_title
+        , author_id as userlist_author_user_id
```

Comment:
For foreign key column, we probably want to keep it the same name as primary key 
```suggestion
        , author_id as user_id
```